libraries/AP_NavEKF2/AP_NavEKF2_Outputs.cpp

```diff
@@ -541,6 +548,9 @@ void NavEKF2_core::send_status_report(mavlink_channel_t chan)
     if (filt_state.flags.pred_horiz_pos_abs) {
         flags |= EKF_PRED_POS_HORIZ_ABS;
     }
+    if (filt_state.flags.gps_glitching) {
+        flags |= (1<<15);
```

Comment:
The flag should be added to MAVLink and then using the name here.

And maybe we should think about sending the generic ESTIMATOR_STATUS, but that's for another time.